docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.  
 ![browse project](img/content_trust.png)
 
-### Vulnerability scanning via Clair 
-**CAUTION: Clair is an optional component, please make sure you have already installed it in your Harbor instance before you go through this section.**
+## Vulnerability Scanning
+
+Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. You can also connect Harbor to additional vulnerability scanners by using an interrogation service. The additional scanners that are currently supported are [Anchore](https://anchore.com/harbor/) and [Aqua](https://github.com/aquasecurity/harbor-scanner-aqua).
```

Comment:
I’m creating the compatibility list which includes the supported 3rd scanners. I think here we should refer to that doc.

The doc is almost done and ready for your reviews. @stuclem @xaleeks

Reply:
please link it over here when this is merged @stuclem

Reply:
Done in https://github.com/goharbor/harbor/pull/10029/commits/40e41d0f39c801cea141d7543c3d6c9307d659ac.